apps/explorer/components/Transaction/index.tsx

```diff
@@ -87,33 +95,33 @@ export function Transaction({ title, transaction }: Props) {
       return []
     }
     const operations: EntityListItemProps[] = []
-    transaction.storage_contracts?.forEach((contract) => {
+    transaction.fileContracts?.forEach((contract) => {
       operations.push({
         label: 'contract formation',
         type: 'contract',
         href: routes.contract.view.replace(':id', contract.id),
         hash: contract.id,
       })
     })
-    transaction.contract_revisions?.forEach((contract) => {
+    transaction.fileContractRevisions?.forEach((contract) => {
       operations.push({
         label: 'contract revision',
         type: 'contract',
         href: routes.contract.view.replace(':id', contract.id),
         hash: contract.id,
       })
     })
-    transaction.host_announcements?.forEach((host) => {
-      operations.push({
-        label: 'host announcement',
-        // type: 'host',
-        hash: host.net_address,
-      })
-    })
-    transaction.storage_proofs?.forEach((proof) => {
+    // transaction.host_announcements??.forEach((host) => {
```

Comment:
Will explored not have these?

Reply:
We did not get an answer on this, all good for this PR, but maybe record a todo on this and we can follow up on the discussion thread.